fastlane/Fastfile

```diff
@@ -1,13 +1,34 @@
 fastlane_version '2.53.1'
 
 before_all do
-  ensure_git_branch
+  ensure_git_branch(
+    branch: 'release'
+  )
   ensure_git_status_clean
-  git_pull
 end
 
 platform :ios do
-   # iOS Lanes
+
+  before_all do
+    username = ENV["APPLE_ID"]
```

Comment:
We could have a .env file that's ignored by VCS where you can supply env variables, then you don't need to write it every time you run the deploy cmd